keytableprinter/keytableprinter.go

```diff
@@ -2,94 +2,179 @@ package keytableprinter
 
 import (
 	"fmt"
+	"sort"
 	"strings"
 
 	"github.com/fluidkeys/fluidkeys/colour"
 	"github.com/fluidkeys/fluidkeys/pgpkey"
 )
 
+type column = []string
+type row = []string
+
 const gutter = "  "
+const divider = "---"
 
-var headers = [3]string{
-	"Email address",
-	"Created",
-	"Next rotation",
+var header = row{
+	colour.Info("Email address"),
+	colour.Info("Created"),
+	colour.Info("Status"),
 }
 
+var placeholderDividerRow = row{divider, divider, divider}
+
 func Print(keys []pgpkey.PgpKey) {
-	fmt.Printf("%s\n", makeTable(keys))
+	rows := makeTableRows(keys)
+	rowStrings := makeStringsFromRows(rows)
+	for _, rowString := range rowStrings {
+		fmt.Println(rowString)
+	}
 }
 
-func makeTable(keys []pgpkey.PgpKey) string {
-	var table string
-	table += fmt.Sprintf("%s\n", colour.LightBlue(makeHeader(columnWidths(keys))))
-	table += fmt.Sprintf("%s\n", makeHorizontalUnderlines(columnWidths(keys)))
+func makeTableRows(keys []pgpkey.PgpKey) []row {
+	var rows []row
+	rows = append(rows, header)
+	rows = append(rows, placeholderDividerRow)
```

Comment:
intriguing, I thought `placeholderDividerRow` contained three dashes per line e.g. "---     ---    ---", but I don't think that's how the table ends up, so I'm interested to see where this is going!